src/l500/l500-options.h

```diff
@@ -94,24 +99,39 @@ namespace librealsense
         void set_read_only( bool read_only );
         void set_manually( bool set );
 
-    private:
+    protected:
         float query_default( bool & success ) const;
-        
 
         l500_control _type;
-        l500_device* _l500_dev;
+        l500_options* _owner;
         hw_monitor* _hw_monitor;
         option_range _range;
         uint32_t _width;
```

Comment:
Are these two in use? Remove?

Reply:
I replaced l500_device* with l500_options*

Reply:
I meant _width and _height

Reply:
removed